x/group/spec/group_policy.md

```diff
@@ -0,0 +1,88 @@
+# Group Policy Specification
+
+## Types
+
+### `GroupPolicyID`
+
+```go
+type GroupPolicy uint64 
+```
+
+The ID of a group policy, generated as an auto-incrementing integer.
+
+## Messages
+
+### `MsgCreateGroupPolicy`
+```go
+// MsgCreateGroupPolicy creates a new group policy with the provided
+// DecisionProtocol. A group policy allows groups to set different decision
+// protocols for different types of action.
+type MsgCreateGroupPolicy struct {
```

Comment:
q: can you elaborate how this is different to `MsgCreateGroup` and how it is used?